scalableminds/webknossos
# Implement option to export proofreading as segmentation

### Steps to test:
- In a freshly created Annotation (with agglomerate file mapping) use the proofreading tool and merge the volume annotation with its fallback layer (by clicking onto the three dots next to the desired segmentation layer)
- In the resulting Dataset the changes should be applied
- Other ways of materializing volume annotations should still work as before and could also be tested

[corresponding voxelytics pr](https://github.com/scalableminds/voxelytics/pull/3859)

------
